src/popup/pages/TransactionDetails.vue

```diff
@@ -409,5 +417,9 @@ export default defineComponent({
       color: variables.$color-warning;
     }
   }
+
+  .payload {
+    margin-top: 0;
+  }
```

Comment:
As I mentioned in one of the previous comments - the margin top 0 should be the default state for the payload component.